modular_skyrat/modules/hydra/snout.dm

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
why this

Reply:
I actually dont know it was 5am

Reply:
if it's not working, don't add it into code commented out